documentation/src/main/asciidoc/topics/ref_jgroups_system_properties.adoc

```diff
@@ -89,7 +84,7 @@ The following system properties apply to `default-jgroups-ec2.xml`:
 
 .Kubernetes
 
-The following system properties apply to `default-jgroups-kubernetes.xml`:
+The following system properties apply only to `default-jgroups-kubernetes.xml`:
 
 [%header,cols="1,3,2,1"]
 |===
```

Comment:
where is `dns_query` come from? It is mentioned in the column "required".

Reply:
`dns_query` is set in DNS_PING. I've updated this description. great catch btw!